test/bdd/docker.go

```
@@ -0,0 +1,87 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package bdd
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// DockerHelper helper for docker specific functions
+type DockerHelper interface {
+	GetIPAddress(containerID string) (string, error)
+	RemoveContainersWithNamePrefix(namePrefix string) error
+}
+
+// NewDockerCmdlineHelper returns a new command line DockerHelper instance
+func NewDockerCmdlineHelper() (DockerHelper, error) {
+	dockerCmdlineHelper := &dockerCmdlineHelper{}
+	return dockerCmdlineHelper, nil
```

Do we need to return an error here?

---

done